### PR TITLE
feat: added reference for content assignment list endpoint and wrote test cases

### PR DESCRIPTION
**Pull request by sameeramin**

### CODE REVIEW

File: tests/test_enterprise_access.yaml [hunk 1]
```diff
@@ -23,6 +23,19 @@
...
+    - method: 'GET'
+    - headers: { 'Authorization': 'aeiou', 'Content-Type': 'application/json' }
+    - expected_status: [200]
+
+- test:
+    - name: 'Assignment configuration list request rejected without authorization'
+    - url: '/enterprise/v1/assignment-configurations/0123456789abcdefg/admin/assignments/'
+    - method: 'GET'
+    - expected_status: [400]
+
```
Comment (sameenfatima78) [inline tests/test_enterprise_access.yaml:38]: can we add one more test with any of the valid query params and then one with an invalid param?

**Review by sameenfatima78 — APPROVED** [review]
Looks good, just one small suggestion.

File: tests/test_enterprise_access.yaml (hunk 1 above)
Comment (sameeramin) [reply tests/test_enterprise_access.yaml:38]: Sure 🫡

Comment (sameeramin) [reply tests/test_enterprise_access.yaml:45]: @sameenfatima78 added one more test please have a look

**Pull request merged**